addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -5470,9 +5011,9 @@ stock void AnglesToVelocity(float fAngle[3], float fVelocity[3], float fSpeed =
 	fVelocity[0] = Cosine(DegToRad(fAngle[1]));
 	fVelocity[1] = Sine(DegToRad(fAngle[1]));
 	fVelocity[2] = Sine(DegToRad(fAngle[0])) * -1.0;
-
+	
```

Comment:
`AnglesToVelocity(float fAngle[3], float fVelocity[3], float fSpeed = 1.0)` should be `AnglesToVelocity(float vecAngle[3], float vecVelocity[3], float vecSpeed = 1.0)`